packages/dotcom-build-base/package.json

```diff
@@ -33,7 +33,7 @@
     "node": ">= 14.0.0",
     "npm": "7.x || 8.x"
   },
-  "files": ["dist/"],
+  "files": ["dist/", "src/"],
```

Comment:
**question**: why does `src` need to be included here? it just includes a typescript file which is transpiled into the `dist` directory

Reply:
This was to replicate what was published when using Athloi. We were previously using `.npmignore` to ignore all files we didn't want which were related to tests and `node_modules` which meant `src` was published. For this, we have made the published packages at parity with the previous publish workflow but we should do work in the future to evaluate what we actually need to publish for each package.